webapp/locales/en.json

```diff
@@ -420,5 +420,8 @@
+  },
+  "changelog": {
+    "text": "<p><strong>All referenced changes refer to the new development version \"Nitro\", not the version running here, which is in freeze.</strong></p><br /><p><strong>March2019</strong></p><ul><li>Dialog for password change has been added.</li><li>An editor for writing posts has been added.</li><li>A dialogue for blocking content which violates the terms of service had been added for the moderators.</li><li>The \"Shout\" button has been added to the article view.</li><li>All software repositories on Github have been merged into a big one so that we don't suffer from independent version information and incompatible interfaces anymore.</li></ul><p><strong>February 2019</strong></p><ul><li>The automated import of data from the Alpha-version to the Nitro had been tested with test data</li><li>The web interface of the Nitro had been testet for correct display on mobile devices.</li></ul><p><strong>January 2019</strong></p><ul><li>A Button for reporting critical content has been added</li><li>An moderation backend for reported content has been implemented</li><li>A Bug with the user authentication has been fixed</li><li>User can input country, region or city and it is her / his decision how accurate it should be</li><li>User settings field for setting the interface and preferred content language</li><li>Integration of Nuxt-i18n toolkit for support of serverside rendered pages of webapps</li><li>Integration of Lokalise.co pipeline for translations</li><li>Updated used toolkits and frameworks to newest versions</li></ul><p><strong>December 2018</strong></p><ul><li>[in progress] Add icons to categories and tagging (how many users are using this tag, not how often the tag is used generelly) </li><li>Updated to actual eslint version for source code quality</li><li>[in progress] Coding a blacklist for the Nitro for hiding unwanted user's content</li></ul><p><strong>November 2018</strong></p><ul><li>Coding a functionality, so that users, which had not donated yet, later get a chance to do so</li><li>Coding a database query to identify users, which had not completed the registration dialog</li><li>Introdution of a versioning scheme to identify the running social network code version</li><li>Login for the new Nitro version with a nice login dialogue.</li><li>Moved project planning to Zenhub</li><li>Decided to use srum for project management</li></ul><p>October 2018</p><ul><li>Started work on \"Nitro\". The Nitro-version replaces the old Database by a <a class=\"hc-editor-link-blot\" href=\"https://www.youtube.com/watch?v=5Tl8WcaqZoc\" target=\"_blank\">new graph database</a> (external link to youtube) called neo4j. That means that our future development is much easier, than before.</li></ul>"
```

Comment:
```suggestion
    "text": "<ul> <li>Dialog for password change has been added.</li> <li>An editor for writing posts has been added.</li> <li>A dialogue for blocking content which violates the terms of service had been added for the moderators.</li> <li>The \"Shout\" button has been added to the article view.</li> <li>All software repositories on Github have been merged into a big one so that we don't suffer from independent version information and incompatible interfaces anymore.</li> </ul> <p><strong>February 2019</strong></p> <ul> <li>The automated import of data from the Alpha-version to the Nitro had been tested with test data</li> <li>The web interface of the Nitro had been tested for correct display on mobile devices.</li> </ul> <p><strong>January 2019</strong></p> <ul> <li>A Button for reporting critical content has been added</li> <li>An moderation backend for reported content has been implemented</li> <li>A Bug with the user authentication has been fixed</li> <li>User can input country, region or city and it is her / his decision how accurate it should be</li> <li>User settings field for setting the interface and preferred content language</li> <li>Integration of Nuxt-i18n toolkit for support of serverside rendered pages of webapps</li> <li>Integration of Lokalise.co pipeline for translations</li> <li>Updated used toolkits and frameworks to newest versions</li> </ul> <p><strong>December 2018</strong></p> <ul> <li>[in progress] Add icons to categories and tagging (how many users are using this tag, not how often the tag is used generally)</li> <li>Updated to actual eslint version for source code quality</li> <li>[in progress] Coding a blacklist for the Nitro for hiding unwanted user's content</li> </ul> <p><strong>November 2018</strong></p> <ul> <li>Coding a functionality, so that users, which had not donated yet, later get a chance to do so</li> <li>Coding a database query to identify users, which had not completed the registration dialog</li> <li>Introdution of a versioning scheme to identify the running social network code version</li> <li>Login for the new Nitro version with a nice login dialogue.</li> <li>Moved project planning to Zenhub</li> <li>Decided to use srum for project management</li> </ul> <p>October 2018</p> <ul> <li>Started work on \"Nitro\". The Nitro-version replaces the old Database by a <a class="\&quot;hc-editor-link-blot\&quot;" href="\&quot;https://www.youtube.com/watch?v=5Tl8WcaqZoc\&quot;" target="\&quot;_blank\&quot;">new graph database</a> (external link to youtube) called neo4j. That means that our future development is much easier, than before.</li> </ul>"
```